src/vm/diagnosticsprotocol.h

```diff
@@ -49,6 +56,394 @@ bool TryParseString(uint8_t *&bufferCursor, uint32_t &bufferLen, const T *&resul
     return true;
 }
 
+namespace DiagnosticsIpc
+{
+    enum class IpcVersion : uint8_t
+    {
+        DOTNET_IPC_V1 = 0x01,
+        // FUTURE
+    };
+
+    enum class DiagnosticServerCommandSet : uint8_t
+    {
+        // Debug         = 0x00,
```

Comment:
Let's leave 0 **never** used? It looks like this can prevent accidental usage for carelessly written clients.

Reply:
Are you suggesting we use 0 for a value in this case?  I've modified it to mark it as reserved rather than debug since that is a more apt description.

Reply:
Sounds good to me